pkg/cmd/secvulnMaven.go

```diff
@@ -5,33 +5,319 @@
 package cmd
 
 import (
+	"encoding/xml"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
 
 	"github.com/spf13/cobra"
 )
 
 var (
-    secvulnMavenCmd = &cobra.Command{
+	secvulnMavenCmd = &cobra.Command{
 		Use:   "maven",
 		Short: "Generate psuedo maven project for security vulnerability scanning",
 		Long:  "Generate psuedo maven project for security vulnerability scanning",
 		Run:   secvulnMavenExecute,
 	}
 
-	secvulnMavenParentDir     string
-	secvulnMavenPomUrls    *[]string
+	secvulnMavenParentDir string
+	secvulnMavenPomUrls   *[]string
+	secvulnMavenPomRepos  *[]string
+
+	completedProjects []string
```

Comment:
why not use Dependency instead of String,   to simplify the code later so you don't have to search for versions etc